package.json

```diff
@@ -80,10 +80,10 @@
     "sort-package-json": "^1.53.1",
     "source-map-support": "^0.5.21",
     "ts-node": "^10.9.2",
-    "typedoc": "^0.17.8",
+    "typedoc": "^0.17.7",
     "typedoc-plugin-external-module-map": "1.2.1",
     "typedoc-plugin-nojekyll": "^1.0.1",
-    "typescript": "~5.1.0"
```

Comment:
Originally upgraded to fix a linting issue, but the linting issue is not appearing any longer so dropping it down to keep it in alignment with commerce-sdk-isomorphic